### DISCUSSION TITLE
Software I need for icon theme creation.

### DISCUSSION THREAD

**Discussion by nestoris** [nestoris/Win98SE#33] [Ideas]
Dear developers and coders! I know some script languages (gawk, a bit of javascript) and fiew frameworks (gtkdialog, zenity/yad, html) but it's not enough.

I need:
* A GUI application for setting up theme for toggling style of icons (win98/win2k, smooth/sharp, dinosaur/fox-on-globe etc.)
* A GUI application for finding deadlinks in subfolders, icons of inappropriate size etc.
* Plugin for GIMP or **Krita** for realtime displaying of multilayer image in 256-color with diffusion and/or pattern blending of colors. **IS DONE! THANK YOU DEVELOPERS!**
* And if you help me to finish [my application for editing icon theme](https://github.com/nestoris/icon-theme-tool), or rewrite it on another language and framework.

That's actually not all. If you are a coder and can make front-ends, please write here.
I will be glad to ask you for help and it will motivate me to work on the icon theme.

**Comment by Dejweed** [comment]
For GUI toggling something like [Chicago95 installer](https://github.com/grassmunk/Chicago95/blob/master/installer.py) or [PlusGUI.py](https://github.com/grassmunk/Chicago95/tree/master/Plus#run_GUI) could be used, but much simpler.
